eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -113,10 +110,10 @@ jobs:
         displayName: 'Publish unit test results'
 
       - script: |
-          echo "Getting list of projects in scheme ${{parameters.ServiceDirectory}}"
+          echo "Getting list of projects in scheme 'ci'"
```

Comment:
This should probably print the actual scheme i.e. "all"

Reply:
Actually is there a reason we don't just keep this file as-is, rename "ServiceDirectory" to "scheme", and pass in "scheme: all" in the ci.yml? It seems like it would let us keep the ability to run against other schemes in the future (e.g. maybe for a manual verification build of some sort).